spec/features/incoming_mail_handler_spec.rb

```diff
@@ -49,21 +50,25 @@
     mandrill_event = mandrill_payload_from_message(mail)
     mandrill_event[0]['msg']['from_email'] = user.email_address
     handler = IncomingMail::Handler.new
-    expect(my_approval.proposal.existing_observation_for(user)).to be_falsey
-    expect(my_approval.proposal.existing_approval_for(user)).to be_falsey
+
+    expect(my_approval.proposal.existing_observation_for(user)).not_to be_present
+    expect(my_approval.proposal.existing_approval_for(user)).not_to be_present
+
     resp = handler.handle(mandrill_event)
+
     expect(resp.action).to eq(IncomingMail::Response::COMMENT)
-    expect(resp.comment.proposal.existing_observation_for(user)).to be_truthy
-    expect(resp.comment.proposal.existing_approval_for(user)).to be_falsey
+    expect(resp.comment.proposal.existing_observation_for(user)).to be_present
+    expect(resp.comment.proposal.existing_approval_for(user)).not_to be_present
     expect(my_approval.user).to_not eq(my_proposal.individual_approvals.last.user)
-    recipients = [
-      user.email_address, # observer notification
```

Comment:
This feels like a UX issue to me. If, as a non-observer, I create a comment by replying to an email (admittedly an edge case), then as it is now, I will get zero emails confirming that I made the comment. Maybe that's ok (it is what GitHub does, e.g.). I don't know. I just wanted to point out that the current behavior (multiple emails) will change to zero emails.